test/metabase/query_processor_test/alternative_date_test.clj

```diff
@@ -225,15 +228,33 @@
                  (mt/rows (mt/dataset string-times
                             (qp/process-query
                               (assoc (mt/mbql-query times)
-                                     :middleware {:format-rows? false})))))))))
+                                     :middleware {:format-rows? false}))))))))
+
+      (testing "mongo only supports datetime"
+        (mt/test-drivers #{:mongo}
+          (is (= [[(t/instant "2004-10-19T10:23:54Z")]
+                  [(t/instant "2008-10-19T10:23:54Z")]
+                  [(t/instant "2012-10-19T10:23:54Z")]]
+               (mt/rows (mt/dataset string-times
```

Comment:
It's slightly preferrable to put `mt/dataset` around the entire test because then it adds `testing` context. But it's not that important. For next time tho you might want to consider doing it